float64 back_left_wheel # back left wheel speed in m/s
float64 back_right_wheel # back right wheel speed in m/s
float64 front_left_wheel # front left wheel speed in m/s
float64 front_right_wheel # front right wheel speed in m/s
